articles/150_ros_command_line_arguments.md

```diff
@@ -167,16 +167,25 @@ another_node:
      string_param: bar
 ```
 
-#### Logging level
+#### Logging level assignments
 
-Minimum logging level can be externally set using the `--log-level` option.
+Minimum logging level can be externally set either globally or per logger using the `--log-level` option.
 
-As an example, to set logging level to `DEBUG` for `some_ros_executable`, one may execute:
+As an example, to set a global logging level to `DEBUG` for `some_ros_executable`, one may execute:
 
 ```sh
 ros2 run some_package some_ros_executable --ros-args --log-level DEBUG
 ```
 
+Some specific loggers can be set using the `--log-level` option as well:
```

Comment:
So does `Some` here mean there are loggers you cannot set this way? If not, I'd drop it. If so, then I think we need some clarification on that.